droolsjbpm-bpms-distribution/pom.xml

```diff
@@ -77,6 +83,10 @@
           <groupId>org.glassfish.jaxb</groupId>
           <artifactId>jaxb-runtime</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.jboss.spec.javax.transaction</groupId>
+          <artifactId>jboss-transaction-api_1.2_spec</artifactId>
```

Comment:
if we are moving to jackarta namespace, why exists this exclusion ?
jakarta.transaction-api

Reply:
I guess something in jbpm-test brings it transitively. Maybe it can be removed, but maybe not.

Reply:
@mbiarnes you removed https://github.com/kiegroup/jbpm/blob/master/jbpm-test/pom.xml#L114-L118 in your PR in jbpm repository https://github.com/kiegroup/jbpm/pull/1917/files#diff-a8586b3b636a6c20c9967fe2b1d8b8000b7d75d8c0a6ae47a4d2c7ba7c557175R127. So this should not be needed, but we can wait on stabilizing of PR set and get rid off it?